components/renku_data_services/project/api.spec.yaml

```diff
@@ -305,6 +305,114 @@ paths:
           $ref: "#/components/responses/Error"
       tags:
         - projects
+  /projects/{project_id}/secret_slots:
```

Comment:
suggestion: We should be consistent with the names. We call these `secret_slots` here in the path but further below there is `session_secret_slots`. I think it is clear enough if we just use `secret_slots` everywhere. Or regardless of what we decide we should use the same term across the board.

Reply:
Sure, though I am a bit afraid about having more types of secret slots in the future.

Reply:
The API is updated in 6155cf330ac226353e7f3f58523a322d9e47ac50: the endpoint is now `/projects/{project_id}/session_secret_slots` for consistency.